src/coreclr/jit/gentree.cpp

```diff
@@ -6957,8 +6957,9 @@ GenTreeIntCon* Compiler::gtNewStringLiteralLength(GenTreeStrCon* node)
         return gtNewIconNode(0);
     }
 
-    int             length = -1;
-    const char16_t* str    = info.compCompHnd->getStringLiteral(node->gtScpHnd, node->gtSconCPX, &length);
+    constexpr int maxStrSize      = 1024;
```

Comment:
nit: this (and https://github.com/dotnet/runtime/blob/b5381bf/src/coreclr/jit/importer_vectorization.cpp#L148) can be just `const int`

Reply:
Thanks! will change if there will be more change requests to avoid re-running CI just for a code-style change 🙂